nvflare/private/fed/app/server/runner_process.py

```diff
@@ -129,8 +133,27 @@ def main():
         raise e
 
 
+def ternimate_process():
+    pid = os.getpid()
+    kill_child_processes(pid)
+    os.killpg(os.getpgid(pid), 9)
+
+
+def kill_child_processes(parent_pid, sig=signal.SIGTERM):
+    try:
+        parent = psutil.Process(parent_pid)
+    except psutil.NoSuchProcess:
+        return
+    children = parent.children(recursive=True)
+    for process in children:
+        process.send_signal(sig)
+
+
```

Comment:
we are not using these two methods, why add them back

Reply:
cleaned up.